packages/link/package.json

```diff
@@ -15,7 +15,7 @@
     "build:js": "cross-env ESM=true babel --root-mode upward src --out-dir dist/esm",
     "build:watch": "npm run build:js -- --watch",
     "prepublish": "npm run build",
-    "storybook": "start-storybook -p 6006",
+    "storybook": "start-storybook -p 6006 -s ./dist",
```

Comment:
are these storybook changes supposed to be committed. i generally don't run build along with storybook so this would be confusing when making changes

Reply:
Yes, that was my intent.

I always build and thought one had to build.  But I guess things have changed since that habit was formed.  

I don't think there's another way to have this kind of test (testing output stylesheets) (which arguably we should not have), but I am seeing a bit of value with it.  In this case, the test case also involves interaction of hover, which is unsnapshottable, so it's less helpful.  But it at least provides an interactive environment to test and find the bug.

Reply:
Jake's right about this one. Without building glamour run on src doesn't output the same and hover appears to work. We might want to refactors our other stories.

Reply:
I think the other stories can remain as-is for now because they're all using React components in the output.

This is the first story that I'm aware of where we're using the CSS stylesheet that's built inside of storybook.

Reply:
I'm not sure the storybook thing is it. I just ran this branch using a build off master and noticed it was working on my machines. I haven't yet run a build with your css mod.

Reply:
Edward, not sure what you mean, but if you want to see the problem in action, try:

https://codesandbox.io/s/dreamy-noether-mnezl

And then to verify the fix,:

the PR should look good after a build and npm run storybook

Reply:
Yea the codepen makes sense to me. 

I mean I wanted to verify the storybook change was necessary and so I first ran build of the entire DS on the master branch. Then I checked out your branch and noticed that when I run your new storybook command I'm not seeing the bug that's in the codpen link.